internal_ws/ykcompile/src/lib.rs

```diff
@@ -52,9 +52,9 @@ lazy_static! {
                                      R10.code(), RBX.code(), R12.code(), R13.code(), R14.code(),
                                      R15.code()];
 
-    // The trace inputs/outputs are always allocated to this reserved register.
+    // The interpreter context is always allocated to this reserved register.
     // This register should not appear in REG_POOL.
-    static ref TIO_REG: u8 = RDI.code();
+    static ref TCTX_REG: u8 = RDI.code();
```

Comment:
Just `CTX_REG`?

Reply:
I agree that `TCTX` is wrong. Maybe `ICTX_REG`? I'm neutral.

Reply:
I did consider proposing the latter too. It feels like this need more (no pun intended) *context* as it's a global thing.

Reply:
Agreed. Fixed in https://github.com/softdevteam/yk/pull/222/commits/685c7ce90fad226764cf093013fdcf52bd7df0c2.